server/app/services/geo/esri/EsriClient.java

```diff
@@ -0,0 +1,88 @@
+package services.geo.esri;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigList;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import play.libs.ws.*;
+import services.geo.AddressAttributes;
+import services.geo.AddressLocation;
+
+public class EsriClient implements WSBodyReadables, WSBodyWritables {
```

Comment:
suggestion (blocking):
```suggestion
public final class EsriClient implements WSBodyReadables, WSBodyWritables {
```